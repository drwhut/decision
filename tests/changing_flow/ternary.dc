Start~#1

< true literal >
Ternary(true, 1, 0)~#2
Print(#1, #2)~#3

< false literal >
Ternary(false, 1.0, 0.5)~#4
Print(#3, #4)~#5

< true non-literal >
MoreThan(10, 5)~#6
Ternary(#6, "10 > 5", "10 <= 5")~#7
Print(#5, #7)~#8

< false non-literal >
LessThan(10, 5)~#9
Ternary(#9, true, false)~#10
Print(#8, #10)~#11

< values being passed in from other nodes >
Add(10, 10)~#12
Multiply(#12, 2)~#13
Div(#12, 2)~#14
Equal(#12, 20)~#15
Ternary(#15, #13, #14)~#16
Print(#11, #16)~#17