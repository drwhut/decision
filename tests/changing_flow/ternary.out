1
0.5
10 > 5
false
40
